Ex1/main.java

```diff
@@ -63,4 +65,42 @@ public static void main(String[] args) {
             }
         }
     }
+    	private static void reverseOrder(ArrayList<contact> phone_book2) {
```

Comment:
The name can be phone_book and not phone_book2, it's a local variable inside the function and also it's a static class variable and not a global variable so nothing will be mixed.